lib/model/security/user.ts

```diff
@@ -87,16 +87,16 @@ export class User {
     }
 
     // Every target must be allowed by at least one profile
-    return this.areTargetsAllowed(profiles, targets);
+    return this.areTargetsAllowed(request, profiles, targets);
   }
 
   /**
    * Verifies that every targets are allowed by at least one profile,
    * while skipping the ones that includes a wildcard since they will be expanded
    * later on, based on index and collections authorized for the given user.
    */
-  private async areTargetsAllowed (profiles: Profile[], targets: Target[]) {
-    const profilesPolicies = await Bluebird.map(profiles, profile => profile.getAllowedPolicies());
+  private async areTargetsAllowed (request: KuzzleRequest, profiles: Profile[], targets: Target[]) {
+    const profilesPolicies = await Bluebird.map(profiles, profile => profile.getAllowedPolicies(request));
```

Comment:
`Bluebird.map` should be replaced by `Promise.all`, it would have detected the error